packages/remix-forms/src/createForm.tsx

```diff
@@ -292,10 +300,14 @@ function createForm({
       ],
     )
 
-    const fieldErrors = (key: keyof SchemaType) => {
-      const message = (formErrors[key] as unknown as FieldError)?.message
-      return browser() ? message && [message] : errors && errors[key]
-    }
+    const fieldErrors = React.useCallback(
```

Comment:
Why use a `useCallback`? Have you considered extracting the function out of the component?